creator-node/src/routes/files.js

```diff
@@ -148,8 +149,11 @@ const getCID = async (req, res) => {
 
         // Set length to be end - start + 1 so it matches behavior of fs.createReadStream
         stream = req.app.get('ipfsAPI').catReadableStream(
-          CID, { offset: start, length: end - start + 1 }
+          CID, { offset: start, length: (end || stat.size) - start + 1 }
```

Comment:
from line 144 above, it seems that end must be at most `stat.size - 1`, so is this a drop in replacement here? shouldn't it be `end - start + 1` or `stat.size - start` as presumably `stat.size` already equals `end + 1`?

Reply:
Thanks for catching this :) i owe you two off by 1s on this.
Tested again and compared results directly to requesting partial content from imgur in postman